modules/editor/webhook/WebhookControls.tsx

```diff
@@ -68,6 +61,12 @@ export function WebhookControls(props: WebhookControlsProps) {
     }
   }, [])
 
+  const saveLabel = () => {
+    if (editorManager.messages.every(m => !m.url)) return "Send"
+    if (editorManager.messages.every(m => m.url)) return "Edit"
+    return "Send/Edit"
```

Comment:
Using slashes disrupts reading flow, I think "Submit" fits better
```suggestion
    return "Submit"
```